src/Cake.Web/App_Data/docs.xml

```diff
@@ -35,7 +35,7 @@
     <item title="Visual Studio Code" file="editors/vscode.md" />
   </section>
   <section id="build-systems" title="Build Systems">
-    <item title="Visual Studio Online" file="build-systems/vso.md" hidden="true"/>
```

Comment:
Not sure we can change the URI of this one since we link to it from the marketplace.